tests/automember/test_automember_client_context.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: Test automember
+  hosts: ipaclients, ipaserver
```

Comment:
Do we need `ipaserver` here? It is not in all the `test_*_client_context.yml` files.

Reply:
When using both, you can this file against a client and a server, using something similar to the [example inventory file for a cluster](https://github.com/freeipa/ansible-freeipa/blob/master/inventory/hosts.cluster).

Reply:
I added both host list to all `*_client_context.yml` files.